src/test/scala/outwatch/OutwatchSpec.scala

```diff
@@ -1,20 +1,22 @@
 package outwatch
 
-
+import scala.language.implicitConversions
+import scala.concurrent.Future
+import cats.effect.IO
 import monix.execution.Ack.Continue
 import monix.execution.ExecutionModel.SynchronousExecution
 import monix.execution.schedulers.TrampolineScheduler
 import monix.execution.{Cancelable, Scheduler}
 import monix.reactive.Observable
 import org.scalajs.dom.{document, window}
 import org.scalatest.{BeforeAndAfterEach, FlatSpec, Matchers}
+import org.scalatest._
 import outwatch.Deprecated.IgnoreWarnings.initEvent
 
-
 trait EasySubscribe {
 
   implicit class Subscriber[T](obs: Observable[T]) {
-    def apply(next: T => Unit)(implicit s: Scheduler): Cancelable = obs.subscribe { t =>
+    def apply(next: T ⇒ Unit)(implicit s: Scheduler): Cancelable = obs.subscribe { t ⇒
```

Comment:
I'd prefer to stay with ascii arrows instead of unicode ones.

Reply:
I'll update it to use ascii

Reply:
@fdietze updated PR to use `<-` instead of `←` and `=>` instead of `⇒` in tests